packages/kit/types/index.d.ts

```diff
@@ -1,7 +1,7 @@
 /// <reference types="svelte" />
 /// <reference types="vite/client" />
 
-import './ambient';
+import './ambient.js';
```

Comment:
This looks stupid but it's what TS wants when using the new resolution algorithm. Luckily this also works for the old one.